TerriaJS/terriajs
# Fix loading of EPSG:4326, WMS 1.3.0 services

### What this PR does

Fixes https://github.com/TerriaJS/terriajs/issues/6510

Currently we pass `crs` and `srs` as request parameters to `WebMapServiceImageryProvider` instead of passing them as constructor options. This has 2 problems:

- Cesium ~overrides the query parameters~ sets `srs` and `crs` query parameter from `options` [here](https://github.com/CesiumGS/cesium/blob/fceb03ade47a0b994581c8f0c2727129b3221945/Source/Scene/WebMapServiceImageryProvider.js#L263) and [here](https://github.com/CesiumGS/cesium/blob/fceb03ade47a0b994581c8f0c2727129b3221945/Source/Scene/WebMapServiceImageryProvider.js#L291) using values from the constructor `options`.
- When not setting `options.crs`, Cesium chooses a default value of `CRS:84` instead of `EPSG:4326` (in 3d mode), and so skips the `bbox` transformations that are required for correctly loading `EPSG:4326` in WMS version v1.3.0. This causes the above bug (inverted bbox for GetMap queries resulting in server returning transparent tiles)

This PR fixes it by correctly passing `crs` and `srs` as constructor `options` instead of as query parameters.

### Test me
  
- Open this [main branch link](http://ci.terria.io/main/#share=s-o66LYvxOLND41CGwHPuUKlfIBG3) that loads a WMS endpoint which only supports EPSG:4326
- Observe that nothing is rendered on the map, `GetMap` requests are send but returns transparent images. This is because WMS 1.3.0 expects a reversed `bbox` for `EPSG:4326` but the reversing does not happen
- Now open the [link for this branch](http://ci.terria.io/pass-wms-crs/#share=s-o66LYvxOLND41CGwHPuUKlfIBG3)
- Observe that the dataset is rendered correctly on the map
- Test other WMS endpoints to ensure nothing else is broken

### Checklist
